tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRConfig.java

```diff
@@ -23,6 +23,7 @@
 import org.unicode.cldr.icu.dev.test.TestFmwk;
 import org.unicode.cldr.icu.dev.test.TestLog;
 import org.unicode.cldr.test.CheckCLDR.Phase;
+import org.unicode.cldr.test.SubmissionLocales;
```

Comment:
this is a spotless change - SubmissionLocales is mentioned in a javadoc comment somewhere.